src/database.ts

```diff
@@ -17,9 +17,12 @@ type OrderBy<T extends Schema> = {
 	direction: "ASC" | "DESC";
 };
 
+type SelectField<T extends Schema> = keyof T;
+
 export type Query<T extends Schema> = {
 	where?: WhereNode<T>[];
 	orderBy?: OrderBy<T>[];
+	selectFields?: SelectField<T>[];
```

Comment:
```suggestion
	select?: SelectField<T>[];
```

Let's name the field `select` to be more consistent with SQL.